examples/launch_tuning_gluonts.py

```diff
@@ -88,7 +88,7 @@
         config_space, max_t=epochs, resource_attr="epoch_no", mode="min", metric=metric
     )
 
-    max_wallclock_time = 3600 if evaluate_trials_on_sagemaker else 600
+    max_wallclock_time = 3000 if evaluate_trials_on_sagemaker else 600 # wall clock time can be increased to 1 hour for more performance
```

Comment:
Why setting a new constant here? Setting 3000 seconds is fine with me but the comment is then not in sync.

Reply:
This is to ensure that the example finishes in less than one hour. The problem was that the example was failing due to credential expiration (the creds are only valid for one hour). If the experiment runs for an hour, then the runtime goes slightly over an hour because there are a few seconds of setup time before the test reaches the example script. 

The comment is for a future reader who might want to run it for longer